src/frontend/sharedComponents/Text.tsx

```diff
@@ -1,15 +1,82 @@
 import * as React from 'react';
-import {Text as RNText, TextProps} from 'react-native';
+import {Text as RNText, StyleProp, TextProps, TextStyle} from 'react-native';
 import {BLACK} from '../lib/styles';
 
+type fontVariant = 'primary' | 'body';
+
+/**
+ *
+ * Font, font sizing and font weight, should follow designs in figma.
+ * Generally all headers, labels, and button us Rubik (set `variant` to "primary").
+ * Paragraph and body text should use the device's System Font.
+ * Refer to this [design sytem](https://www.notion.so/digidem/Typography-a1600603854445ad82b8d67d4067be1a) for more info.
+ */
 export const Text = ({
   children,
   style,
+  variant = 'body',
```

Comment:
Requiring developers to manually set variant="primary" seems error-prone. Maintaining this across so many different components seems unmanageable.